tests/openvino/native/test_node_utils.py

```diff
@@ -60,10 +62,10 @@ def test_is_node_with_bias(model_to_create, is_with_bias, node_name):
 @pytest.mark.parametrize(
     "weights_port_id, transpose, shape, expected_channel_axes",
     [
-        (0, False, (1,), [0]),
+        (0, False, (1,), []),
         (0, True, (1,), []),
         (1, False, (1,), []),
-        (1, True, (1,), [0]),
+        (1, True, (1,), []),
         (0, False, (1, 1), [0]),
```

Comment:
As we found, ["Transpose attributes are ignored for 1D tensors."](https://docs.openvino.ai/2023.0/openvino_docs_ops_matrix_MatMul_1.html), so tests were adjusted accordingly